Widgets/widget-InvestigationActivity.json

```diff
@@ -0,0 +1,24 @@
+{
+  "id": "investigation-activity",
+  "version": -1,
+  "name": "Investigation Activity Timeline",
+  "dataType": "entries",
+  "widgetType": "line",
+  "size": 5,
+  "query": "",
+  "isPredefined": false,
```

Comment:
I think it should be set to true and the user will be able to dup it if he wants.
all other content widgets are set to true ATM